LICENSE

```diff
@@ -35,8 +35,8 @@ are met:
      applicable to covered code created under this License.
 
   6. Redistributions of any form whatsoever must retain the following
-     acknowledgment: "This product includes Xdebug, freely available
-     from <https://xdebug.org/>".
+     acknowledgment: "This product includes Xdebug software, freely
```

Comment:
"software" word add in PHP 3.0.1

Reply:
OK. Was that really why 3.0 was not OK? Or was it the line for Zend Engine (that I of course don't have).